Dockerfile

```diff
@@ -1,7 +1,21 @@
-FROM nfcore/base
-LABEL authors="Kevin Menden" \
-      description="Docker image containing all requirements for nf-core/cageseq pipeline"
+FROM nfcore/base:1.10.2
+LABEL authors="Kevin Menden, Tristan Kast, Matthias Hörtenhuber" \
+      description="Docker image containing all software requirements for the nf-core/cageseq pipeline"
 
+# Install the conda environment
 COPY environment.yml /
-RUN conda env create -f /environment.yml && conda clean -a
-ENV PATH /opt/conda/envs/nf-core-cageseq-1.0dev/bin:$PATH
+RUN conda env create --quiet -f /environment.yml && conda clean -a
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/nf-core-cageseq-1.0.0/bin:$PATH
+# Dump the details of the installed packages to a file for posterity
+RUN conda env export --name nf-core-cageseq-1.0.0 > nf-core-cageseq-1.0.0.yml
+# Setup paraclu
```

Comment:
Does this custom installation work with `-profile conda`?

Reply:
hmm, `-profile conda` seems to have a problem before that on macos, because sortmerna 4.2.0 is only available for linux...can I drop conda support 😜

Reply:
you were of course correct, this doesn't work for `-profile conda`. Now trying to get the paraclu script on bioconda. Or is there another pipeline which could get around this issue another way?

Reply:
I don't think there is. In order to support Conda, Docker and Singularity properly all top-level packages must be within the Conda environment. Sometimes there is a need to install lower-level packages within the Docker container that are findable on the system when using `-profile conda` but as you have found this won't work here.